tests/Feature/Users/Api/DeleteUserTest.php

```diff
@@ -12,6 +12,28 @@ class DeleteUserTest extends TestCase
 {
 
 
+//    public function testErrorReturnedViaApiIfUserDoesNotExist()
```

Comment:
I think you can do `$this->markTestIncomplete("YOUR CUSTOM message")` here instead of commenting it out - but that's not a big deal.